evap/evaluation/tests/tools.py

```diff
@@ -245,3 +250,84 @@ def assert_no_database_modifications(*args, **kwargs):
             lower_sql = query["sql"].lower()
             if not any(lower_sql.startswith(prefix) for prefix in allowed_prefixes):
                 raise AssertionError("Unexpected modifying query found: " + query["sql"])
+
+
+class LiveServerTest(SeleniumTestCase):
+    external_host = os.environ.get("TEST_HOST", "") or None
+    browser = "firefox"
+    selenium_hub = os.environ.get("TEST_SELENIUM_HUB", "") or None
+    headless = True
+
+    fixtures = [os.path.join(settings.BASE_DIR, "development", "fixtures", "test_with_migrations.json")]
```

Comment:
From the documentation. I'd expect [`serialized_rollback`](https://docs.djangoproject.com/en/5.0/topics/testing/overview/#test-case-serialized-rollback) to solve our problems here -- what's the problem with that?

Reply:
Seems to work